samples/cellular/modem_shell/src/sock/sock.c

```diff
@@ -570,8 +570,6 @@ int sock_open_and_connect(
 		((struct sockaddr_in *)socket_info->addrinfo->ai_addr)->sin_port = htons(port);
 	} else if (family == AF_INET6) {
 		((struct sockaddr_in6 *)socket_info->addrinfo->ai_addr)->sin6_port = htons(port);
-	} else {
-		assert(0);
```

Comment:
By removing this assert we still have a crash if `sock connect -t raw` is given as family will be `inet` and addr_info NULL pointer accessed.

I'll just check that there is address and keep the assert.